src/pages/LEVANTE/LinkUsers.vue

```diff
@@ -0,0 +1,256 @@
+          :value="rawUserFile"
+          show-gridlines
+          :row-hover="true"
+          :resizable-columns="true"
+          paginator
+          :always-show-paginator="false"
+          :rows="10"
+          class="datatable"
+        >
+          <PvColumn v-for="col of columns" :key="col.field" :field="col.field">
+            <template #header>
+              <div class="col-header">
+                <b>{{ col.header }}</b>
+              </div>
+            </template>
+          </PvColumn>
+        </PvDataTable>
+
+        <div class="submit-container">
+          <PvButton
+            :label="activeSubmit ? 'Linking Users' : 'Start Linking'"
+            :icon="activeSubmit ? 'pi pi-spin pi-spinner' : ''"
+            :disabled="activeSubmit"
+            class="bg-primary mb-2 p-3 w-2 text-white border-none border-round h-3rem m-0 hover:bg-red-900"
+            @click="submitUsers"
+          />
+        </div>
+      </div>
+
+      <div v-if="errorUsers.length" class="error-container">
+        <div class="error-header">
+          <h3>Rows with Errors</h3>
+        </div>
+        <PvDataTable
+          ref="errorTable"
+          :value="errorUsers"
+          show-gridlines
+          :row-hover="true"
+          :resizable-columns="true"
+          paginator
+          :always-show-paginator="false"
+          :rows="10"
+          class="datatable"
+        >
+          <PvColumn v-for="col of errorColumns" :key="col.field" :field="col.field">
+            <template #header>
+              {{ col.header }}
+            </template>
+          </PvColumn>
+        </PvDataTable>
+      </div>
+    </section>
+  </main>
+</template>
+
+<script setup>
+import { ref, toRaw } from 'vue';
+import { csvFileToJson } from '@/helpers';
+import { useToast } from 'primevue/usetoast';
+import { useAuthStore } from '@/store/auth';
+import LinkUsersInfo from '@/components/LEVANTE/LinkUsersInfo.vue';
+import PvButton from 'primevue/button';
+import PvColumn from 'primevue/column';
+import PvDataTable from 'primevue/datatable';
+import PvFileUpload from 'primevue/fileupload';
+
+const authStore = useAuthStore();
+const toast = useToast();
+const isFileUploaded = ref(false);
+const rawUserFile = ref([]);
+const columns = ref([]);
+const errorUsers = ref([]);
+const errorColumns = ref([]);
+const activeSubmit = ref(false);
+
+const requiredFields = ['id', 'parentId', 'teacherId'];
+
+const onFileUpload = async (event) => {
+  rawUserFile.value = await csvFileToJson(event.files[0]);
+  columns.value = Object.keys(rawUserFile.value[0]).map((key) => ({
+    field: key,
+    header: key,
+  }));
+
+  const missingColumns = requiredFields.filter((field) => !columns.value.some((col) => col.field === field));
+  if (missingColumns.length > 0) {
+    toast.add({
+      severity: 'error',
+      summary: 'Error: Missing Columns',
+      detail: `Missing columns: ${missingColumns.join(', ')}`,
+      life: 5000,
+    });
```

Comment:
Please use the existing constants for toast configuration. Feel free to introduce an additional one for extended life config